src/routes/github/repository/github-repository-get.ts

```diff
@@ -11,6 +12,11 @@ export const GitHubRepositoryGet = async (req: Request, res: Response): Promise<
 	const repoName = req.query?.repoName as string;
 	const jiraHost = jiraHostLocals || jiraHostParam;
 
+	if (!jiraHost) {
```

Comment:
We should confirm we have the host before making requests where this is a required arg.